File: src/Core.Domain/Models/UserGameModel.cs
using System;
using Core.Domain.Exceptions;

namespace Core.Domain.Models
{
    public class UserGameModel
    {
        public int Id { get; set; }

        public int UserId { get; set; }

        public int OficialGameId { get; set; }

        public int ScoreTeamA { get; set; }

        public int ScoreTeamB { get; set; }

        public int Points { get; set; }

        public UserModel User { get; set; }

        public OficialGameModel OficialGame { get; set; }
    }
}

File: src/Core.Angular/src/app/models/user-game.model.ts
import { User } from "./user.model";
import { OficialGame } from "./oficial-game.model";

export class UserGame {
    public id: number;
    public UserId: number;
    public GameId: number;
    public ScoreTeamA: number;
    public ScoreTeamB: number;
    public Points: number;
    public User: User;
    public OficialGame: OficialGame;
}